src/sentry/incidents/endpoints/serializers/alert_rule_trigger_action.py

```diff
@@ -6,67 +6,75 @@
 logger = logging.getLogger(__name__)
 
 
-@register(AlertRuleTriggerAction)
-class AlertRuleTriggerActionSerializer(Serializer):
-    def human_desc(self, action):
-        # Returns a human readable description to display in the UI
-        priority = (
-            action.sentry_app_config.get("priority")
-            if isinstance(action.sentry_app_config, dict)
-            else ""
-        )
+def human_desc(
+    action_type,
+    target_type,
+    target_identifier,
+    target,
+    target_display=None,
+    action_target=None,
+    priority=None,
+):
+    # Returns a human readable description to display in the UI
+    if priority:
+        priority += " level"
+
+    if action_type == AlertRuleTriggerAction.Type.EMAIL.value:
+        if action_target:
+            if target_type == AlertRuleTriggerAction.TargetType.USER.value:
+                return "Send a notification to " + target.email
+            elif target_type == AlertRuleTriggerAction.TargetType.TEAM.value:
+                return "Send an email to members of #" + target.slug
+    elif action_type == AlertRuleTriggerAction.Type.PAGERDUTY.value:
+        return f"Send a {priority} PagerDuty notification to " + target_display
+    elif action_type == AlertRuleTriggerAction.Type.SLACK.value:
+        return "Send a Slack notification to " + target_display
+    elif action_type == AlertRuleTriggerAction.Type.MSTEAMS.value:
+        return "Send a Microsoft Teams notification to " + target_display
+    elif action_type == AlertRuleTriggerAction.Type.SENTRY_APP.value:
+        return "Send a notification via " + target_display
+    elif action_type == AlertRuleTriggerAction.Type.OPSGENIE.value:
         if priority:
-            priority += " level"
+            return f"Send a {priority} Opsgenie notification to " + target_display
+        return "Send an Opsgenie notification to " + target_display
+    elif action_type == AlertRuleTriggerAction.Type.DISCORD.value:
```

Comment:
Since this is being pulled out and isn't relying on the AlertRuleTriggerAction stuff - should these include `Action.Type` information in addition to AlertRuleTriggerAction?

Reply:
In the other PR I convert the `Action.type` like so: https://github.com/getsentry/sentry/blob/cf1d641537594eacdbac506c507a787ab6dda3da/src/sentry/incidents/endpoints/serializers/workflow_engine_action.py#L21